index.template.php

```diff
@@ -0,0 +1,118 @@
+<?php
```

Comment:
What's with this file? Is this what you were basing the rest of the pages off of?